app/boards/intel_adsp_ace30_ptl.conf

```diff
@@ -0,0 +1,78 @@
+CONFIG_PANTHERLAKE=y
+CONFIG_IPC_MAJOR_4=y
+CONFIG_IPC4_BASE_FW_INTEL=y
+
+CONFIG_COMP_SRC=y
+CONFIG_MM_DRV=y
+CONFIG_COMP_SRC_IPC4_FULL_MATRIX=y
+
+# power settings
+CONFIG_PM=y
+CONFIG_PM_DEVICE=y
+CONFIG_PM_DEVICE_RUNTIME=y
+CONFIG_PM_DEVICE_RUNTIME_EXCLUSIVE=n
+CONFIG_PM_DEVICE_POWER_DOMAIN=y
+CONFIG_PM_POLICY_CUSTOM=y
+CONFIG_ADSP_IMR_CONTEXT_SAVE=y
+
+CONFIG_POWER_DOMAIN=y
+CONFIG_POWER_DOMAIN_INTEL_ADSP=y
+
+# enable Zephyr drivers
+CONFIG_ZEPHYR_NATIVE_DRIVERS=y
+CONFIG_DAI=y
+CONFIG_DAI_INIT_PRIORITY=70
+CONFIG_DAI_INTEL_DMIC=y
+CONFIG_DAI_DMIC_HW_IOCLK=38400000
+CONFIG_DAI_INTEL_DMIC_NHLT=y
+CONFIG_DAI_DMIC_HAS_OWNERSHIP=n
+CONFIG_DAI_DMIC_HAS_MULTIPLE_LINE_SYNC=y
+CONFIG_DAI_INTEL_SSP=y
+CONFIG_ZEPHYR_DP_SCHEDULER=y
+CONFIG_DMA=y
+CONFIG_DMA_INTEL_ADSP_GPDMA=n
+CONFIG_DMA_DW_LLI_POOL_SIZE=50
+CONFIG_INTEL_MODULES=y
+CONFIG_LIBRARY_MANAGER=y
+CONFIG_INTEL_ADSP_TIMER=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_L3_HEAP=y
+CONFIG_RIMAGE_SIGNING_SCHEMA="ptl"
+
+CONFIG_FORMAT_CONVERT_HIFI3=n
+CONFIG_PCM_CONVERTER_FORMAT_S16_C16_AND_S16_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S32_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S24_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S24_C24_AND_S24_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S24_C32_AND_S24_C24=y
+CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S16_C32=y
+CONFIG_LOG=y
+CONFIG_LOG_MODE_DEFERRED=y
+CONFIG_LOG_FUNC_NAME_PREFIX_ERR=y
+CONFIG_LOG_FUNC_NAME_PREFIX_WRN=y
+CONFIG_LOG_FUNC_NAME_PREFIX_INF=y
+CONFIG_LOG_FUNC_NAME_PREFIX_DBG=y
+CONFIG_COMP_VOLUME_WINDOWS_FADE=y
+CONFIG_COMP_UP_DOWN_MIXER=y
+CONFIG_COMP_CHAIN_DMA=y
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=12000
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=38400000
```

Comment:
This line should come out.  This kconfig isn't an app tunable, it's set by the timer driver.  The only reason it's not in devicetree is history (it predates DTS, and in fact the system timer isn't strictly even a Zephyr device struct).  In this case it's benign, since it's just recapitulating the same 38.4MHz value from the driver.  But it's wrong and it won't work to set it here to anything else.

Tick rate is under the app's control, cycle rate is not.

Reply:
We have different value here for FPGA build so its app-tunable in another .conf but this one could be removed because 38.4 is zephyr default for this soc anyway